internal/store/pod.go

```diff
@@ -1034,6 +1034,50 @@ var (
 					}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_pod_overhead_mcpu_cores",
+			Type: metric.Gauge,
+			Help: "The pod overhead in mCPU cores for running a pod.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if p.Spec.Overhead != nil {
+					if cpu, ok := p.Spec.Overhead[v1.ResourceCPU]; ok {
+						ms = append(ms, &metric.Metric{
+							LabelKeys:   []string{"pod", "node"},
+							LabelValues: []string{p.Name, p.Spec.NodeName},
+							Value:       cpu.MilliValue(),
```

Comment:
it is probably best to stay consistent with other CPU metrics if we can, and report it in cores, rather than millicores.  Does that end up reporting a double?  Or does it report an int (which is presumably floored).

Reply:
It returns a double -- I updated to be consistent.